src/abstractdataframe/show.jl

```diff
@@ -579,3 +579,18 @@ function showcols(io::IO, df::AbstractDataFrame) # -> Void
 end
 
 showcols(df::AbstractDataFrame) = showcols(STDOUT, df) # -> Void
+
+using Juno
+import Juno: Inline, LazyTree, Table, Row, strong
+
+const SIZE = 25
```

Comment:
OK. Though showing the full data frame could be quite problematic if it contains one million rows and one hundred columns. Better take a subset first.